detect/detect.go

```diff
@@ -199,19 +199,19 @@ func (d *Detector) AddGitleaksIgnore(gitleaksIgnorePath string) error {
 }
 
 // DetectBytes scans the given bytes and returns a list of findings
-func (d *Detector) DetectBytes(content []byte) []report.Finding {
```

Comment:
@rgmz probably should keep these functions since there are more than a few companies using gitleaks as a library

Reply:
That was on my mind. What's the better option, new alternative `DetectXContext` methods?

Reply:
Unfortunately I think that's the best option. That, or just wait until v9

Reply:
I've updated the code to preserve the original functions. They're now wrappers for the `*Context` methods.